mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -1438,11 +1535,31 @@ struct RewriteAlignedSubByteIntExt : OpRewritePattern<ConversionOpType> {
     // Perform the rewrite.
     Value subByteExt;
     if (isSigned) {
```

Comment:
nit: `conversionOp.getLoc()` is used several times. I think we can declare a variable, which makes code simpler. E.g., `Location loc = conversionOp.getLoc();`